VultisigApp/VultisigApp/Model/Curve.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Curve.swift
+//  VultisigApp
+//
+//  Created by Artur Guseinov on 16.09.2024.
+//
+
+import Foundation
+
+enum Curve {
```

Comment:
We already have a type call `KeyType` , why create a new one here?  you can get it from using `chain.signingKeyType`